pkg/cmd/server/api/types.go

```diff
@@ -154,6 +154,7 @@ var (
 		{DomainName: "quay.io"},
 		// FIXME: Probably need to have more fine-tuned pattern defined
 		{DomainName: "*.amazonaws.com"},
+		{DomainName: "127.0.0.1.nip.io"},
```

Comment:
`*.127.0.0.1.nip.io`?

Reply:
does this let people target the localhost on production servers? this feels like it should be part of devenv setup

Reply:
I don't see any value in this, either.